plugin/test/scrape.go

```diff
@@ -25,14 +25,18 @@ package test
 import (
 	"fmt"
 	"io"
+	"io/ioutil"
 	"mime"
 	"net/http"
+	"reflect"
 	"strconv"
+	"strings"
 
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
-	"github.com/prometheus/common/expfmt"
-
 	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+	"github.com/prometheus/common/model"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
please, let's not "pollute" non-kubernetes code with kubernetes code. You probably also don't actually need sets, but a simple Go `map` (didn't look at the code yet)`